### ISSUE TITLE
Support for iOS executable

### ISSUE THREAD

**Issue by luca992** [icerockdev/moko-resources#310]
I'm trying to add this to the ios target for an experimental multiplatform compose app. Set up like these: [https://github.com/JetBrains/compose-jb/tree/master/experimental/examples](https://github.com/JetBrains/compose-jb/tree/master/experimental/examples). 

But, I'm running into the issue where `kotlin.IllegalArgumentException: bundle with identifier *.*.*.MR not found` is thrown.

So I was reading about this task:

```
"$SRCROOT/../gradlew" -p "$SRCROOT/../" :yourframeworkproject:copyFrameworkResourcesToApp \
    -Pmoko.resources.PLATFORM_NAME=$PLATFORM_NAME \
    -Pmoko.resources.CONFIGURATION=$CONFIGURATION \
    -Pmoko.resources.BUILT_PRODUCTS_DIR=$BUILT_PRODUCTS_DIR \
    -Pmoko.resources.CONTENTS_FOLDER_PATH=$CONTENTS_FOLDER_PATH
```

And it seems like I probably need to add a script like this. But, because the ios app is an executable and not a framework I don't think moko-resources would support this setup at the moment as I would need an equivalent `copyExecutableResourcesToApp`, correct? If so, is there away to work around this for now?

**Comment by Alex009** [comment]
for now i can't say any workaround, but this feature should be implemented, yes.

**Comment by Alex009** [comment]
will be available in 0.20.0
